### PR TITLE
SWL-5166: add tlv for redundant management.

### PR DESCRIPTION
**Pull request by wilmo119**
Reviewer: @kenchiang @shudongz

### CODE REVIEW

**Comment by bsn-abat** [comment]
Find artifact changes from this pull request at https://github.com/floodlight/loxigen-artifacts/tree/PR-646-b1

File: openflow_input/bsn_tlv [hunk 1]
```diff
@@ -1393,3 +1393,8 @@ struct of_bsn_tlv_lossless : of_bsn_tlv {
     uint16_t type == 188;
     uint16_t length;
 };
+
+struct of_bsn_tlv_redundant_mgmt : of_bsn_tlv {
+    uint16_t type == 189;
+    uint16_t length;
+};
```
Comment (shudongz) [inline openflow_input/bsn_tlv]: Is this a boolean or we need to pass the port name?

Comment (wilmo119) [reply openflow_input/bsn_tlv]: This is a boolean. It is meant to be associated with a port key in the port_config gentable.

Comment (shudongz) [reply openflow_input/bsn_tlv]: Makes sense.

**Comment by shudongz** [comment]
LGTM

**Comment by kenchiang** [comment]
ACK

**Comment by bsn-abat** [comment]
ABAT: START_MERGE:  https://jenkins.bigswitch.com/job/loxigen-abat-automerge/38355//console

**Comment by bsn-abat** [comment]
ABAT: ACCEPT:  Successfully merged

In case you want to see the build log, check out :
https://jenkins.bigswitch.com/job/loxigen-abat-automerge/38355//console
/cc

**Pull request closed without merging**